src/librustdoc/html/static/js/main.js

```diff
@@ -353,6 +353,27 @@ function preLoadCss(cssUrl) {
         loadedDescShard: function(crate, shard, data) {
             this.descShards.get(crate)[shard].resolve(data.split("\n"));
         },
+        paramNameShards: new Map(),
+        paramNameResolvers: new Map(),
+        loadParamNames: async function(crate) {
+            if (this.paramNameShards.has(crate)) {
+                return this.paramNameShards.get(crate);
+            } else {
```

Comment:
You can remove this `else` since you already returned. It'll allow to remove one indent level.